src/pudl/transform/ferc714.py

```diff
@@ -234,73 +272,54 @@
 
 RENAME_COLS = {
     "core_ferc714__respondent_id": {
```

Comment:
I used the rename function from the `transform.classes` for this and the `RenameColumns` param. Its not a bad idea to imagine doing these renamings in csv's like the EIA data but instead of years it could be source.

But that'd be a bigger structural lift to use the extractor tooling over here! Right now in the `transform.classes.rename_columns` we are checking if there are columns we are trying to rename that doesn't show up in the data but ya the cross source validation isn't here.

Reply:
This could be a good thing to circle back to if we have time later.